src/components/Comments/ContainerComment.js

```diff
@@ -126,7 +127,8 @@ export const ReplyComment = ({
   navigateToReplyView,
   findCommentAndUpdate,
   onCommentLongPressed = () => {},
-  updateVote
+  updateVote,
+  feedId
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `feedId` prop has been added here as well. Similar to the previous comment, ensure that this prop is properly validated and used within the component. If it's not optional, make sure it's always provided when the component is used.
```diff
+   feedId
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->